File: Potenza/src/Potenza.java
import java.util.Scanner;


public class Potenza {
	
	private static Scanner console;

	public static void main(String[] args){
		int a;
		int b;
		int ris = 1;
		console = new Scanner(System.in);
		System.out.println("Che numero vuoi elevare?: ");
		a = console.nextInt();
		System.out.println("Per quanto lo elevo?: ");
		b = console.nextInt();
		if (a <= 0){
			System.out.println("Inserisci un numero superiore a 0");
			
		}
		else {
			ris =1;
		while(b > 0){
			ris *= a;
			b --;
		}
		}
	
		System.out.println(ris);

	}
}

File: VEcchi/Python/Fatti/Potenza.py
def Potenza(n, k):
    if n <= 0:
        print "Inserire un valore maggiore di 0."
    elif k < 0:
        print "Inserisci un esponente maggiore o uguale a 0."
    else:
        ris = 1
        
        while(k > 0):
            ris = ris * n
            k = k - 1
            
        return ris

n = input("Numero da elevare a potenza: ")
k = input("L'esponente è? ")

print "La potenza e':", Potenza(n, k)
